acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/tls/ClientSideTlsAcceptanceTest.java

```diff
@@ -183,7 +190,7 @@ void missingKeyStoreForEthSignerResultsInEthSignerTerminating(@TempDir Path work
 
   @Test
   void incorrectPasswordForDownstreamKeyStoreResultsInEthSignerTerminating(@TempDir Path workDir)
-      throws IOException {
+      throws IOException, CertificateException, NoSuchAlgorithmException, KeyStoreException {
```

Comment:
nit: reduce just to "Exception" as per others?

Reply:
blerk i thought i did them :/